src/layout/section/slideSection/SlideSection.unit.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { mount, shallow } from 'enzyme'
```

Comment:
Why using enzyme for some tests? You could use rtl throughout the file

Reply:
I had trouble doing what I wanted with RTL (except for hooks since enzyme don't support them).

Even after reading the docs and some articles, I still don't understand how I should test my components with RTL, while I'm more used to work with Enzyme. So after battling a couple of hours with RTL, I switched to Enzyme =/

Reply:
What you did in the last test should be a good starting point for the first four tests.
You already use `getByText`, so you could maybe do a `expect(screen.getByText('Media')).toBeInTheDocument()` instead of `expect(container.text()).toContain('Media')`.
I wouldn't want to have enzyme/rtl in the same file, also all new files should be in RTL

Reply:
Ok, made some changes, thanks for the insight on `toBeInTheDocument()`.